src/components/TimeControl/util.ts

```diff
@@ -601,6 +601,16 @@ export function durationString(seconds): string { /* {{{ */
 
     return ret.trim();
 } /* }}} */
+export function daysOnlyDurationString(seconds): string { /* {{{ */
+    let days = Math.floor(seconds / 86400);
+
+    let ret: string = "";
+    if (days) {
+        ret += days + " " + ngettext(_("Day"), _("Days"), days);
```

Comment:
Two more things here: `ngettext` is a translation function, so we don't translate before we call that - it's just `ngettext("Day", "Days", days)`. And second thing - for translations, we can't assume the order X <translations> is the correct order in every language, so we need to use `interpolate` too

such as `interpolate(_("{{number_of_days}} {{day_or_days_plurality}}"), {number_of_days: days, day_or_days_plurality: ngettext("Day", "Days", days)})`

Reply:
That makes sense. I'll fix that up tomorrow morning. Thanks